src/main/drivers/pwm_mapping.c

```diff
@@ -95,7 +95,7 @@ bool CheckGPIOPinSource(ioTag_t tag, GPIO_TypeDef *gpio, uint16_t pin)
 
 pwmIOConfiguration_t *pwmInit(drv_pwm_config_t *init)
 {
-#ifndef SKIP_RX_PWM_PPM
+#if defined(USE_PWM) || defined(USE_PPM)
```

Comment:
Can you change these to `USE_RX_PWM` and `USE_RX_PPM`. It's more consistent. (I've also got a betaflight PR in the works to do the same there.)